osquery/tables/system/windows/certificates.cpp

```diff
@@ -323,6 +551,29 @@ void enumerateCertStore(const HCERTSTORE& certStore,
 
   if (certContext == nullptr && GetLastError() == CRYPT_E_NOT_FOUND) {
     TLOG << "    Store was empty.";
+
+    // Personal stores for other users come back as empty, even if they are not.
+    if (storeName == "Personal" && !username.empty()) {
+      // Avoid duplicate rows for personal certs we've already inserted up
+      // front.
+      if (storeLocation != "Users" || boost::ends_with(storeId, "_Classes")) {
+        TLOG << "    Trying harder to get Personal store.";
```

Comment:
Generally this pattern of trying to 'prettify' the debug output should be avoided. I see this convention is used else-where in this file. :/ So I am torn.

Reply:
It's like that elsewhere in the file only because of the recently merged #5631, so I don't think changing it should be a problem! I originally added that kind of indentation because it makes the logs for this table much easier to read during development/debugging. I see definitely see why it would be discouraged, though, and would be totally fine changing it.

And on this note anyway, the debug logs added by the aforementioned PR are pretty verbose (there are some in the innermost "loop"/callback), so it might even be best to remove some of them, or change their debug level such that you need to opt in further (beyond `--verbose`) to see them. (not exactly sure if that's possible, or how it would be done).

Reply:
Another issue is the inconsistent use of `TLOG` vs `VLOG`; will fix